sdk/servicebus/azure-messaging-servicebus-stress/templates/job.yaml

```diff
@@ -19,7 +19,8 @@ spec:
           source $ENV_FILE &&
           export CONTAINER_NAME=sender &&
           export APPLICATIONINSIGHTS_ROLE_NAME=sender &&
-          java "org.springframework.boot.loader.JarLauncher" \
+          java -javaagent:BOOT-INF/classes/applicationinsights-agent-3.4.1.jar \
```

Comment:
Oh, I haven't added the agent for sender because there is no custom telemtry data for sending scenario. But yes, we need to add this to send SDK telemtry. And EH stress test also need this if we want to track the sender client data.